### DISCUSSION TITLE
Implementation of Steam Login & RCON servers

### DISCUSSION THREAD

**Discussion by 4NetStudios** [NamelessMC/Nameless#3034] [Ideas]
I know that this CMS is mainly for Minecraft servers, but a nice addon to it would be Steam Login and RCON server stuff.

This implementation would mean that NamelessMC could be used for more than just Minecraft such as CS:GO, Garrys Mod etc.
No need for things like LUA "plugins" since it would use the RCON protocol.

This would also spread the NamelessMC CMS to a larger audience.
I would DEFINETLY use this feature for adding my Garrys Mod server to NamelessMC to integrate the Store I have and have live player count etc directly on the site instead of replying on a 3rd party for it.

**Comment by PadowYT2** [comment]
Probably won't be added at all. The word MC stands for Minecraft

1. Steam Login - it has a lot of limitations as aberdeener says, so won't be implemented
2. There's a module called RconMC, but I don't want to make it for Steam games, as I don't have servers for them

**Comment by tadhgboyle** [comment]
Hi there,
Thanks for the detailed requests.

Unfortunately, steam oauth login is currently not possible for us to
implement - I doubt this will change. There are some technical reasons
behind this.

Can you elaborate more on the rcon support? We are definitely wanting to
move more towards more generic game support!
What would rcon enable a user to do with games like gmod and cs:go?
